### PR TITLE
[ZT] Gateway OR operator

### PR DESCRIPTION
**Pull request by maxvp**

### CODE REVIEW

File: content/cloudflare-one/policies/filtering/dns-policies/_index.md [hunk 1]
```diff
@@ -9,14 +9,17 @@ weight: 2
 
 When a user makes a DNS request to Gateway, Gateway matches the request against the content or security categories you have set up for your organization. If the domain does not belong to any blocked categories, or if it matches an Override policy, the user's client receives the DNS resolution and initiates an HTTP connection.
 
-A DNS policy consists of an **Action** as well as a logical expression that determines the scope of the action. To build an expression, you need to choose a **Selector** and an **Operator**, and enter a value or range of values in the **Value** field.
+A DNS policy consists of an **Action** as well as a logical expression that determines the scope of the action. To build an expression, you need to choose a **Selector** and an **Operator**, and enter a value or range of values in the **Value** field. You can use **And** and **Or** logical operators to evaluate multiple conditions.
 
 - [Actions](#actions)
 - [Selectors](#selectors)
-- [Operators](#operators)
+- [Comparison Operators](#comparison-operators)
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/dns-policies/_index.md]:
```suggestion
- [Comparison operators](#comparison-operators)
```

File: content/cloudflare-one/policies/filtering/dns-policies/_index.md [hunk 2]
```diff
@@ -9,14 +9,17 @@ weight: 2
...
 - [Value](#value)
+- [Logical Operators](#logical-operators)
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/dns-policies/_index.md]:
```suggestion
- [Logical operators](#logical-operators)
```

File: content/cloudflare-one/policies/filtering/dns-policies/_index.md (hunk 1 above)
Comment (ranbel) [reply content/cloudflare-one/policies/filtering/dns-policies/_index.md]: We use sentence case unless it's a product name. I know the dash is not always consistent with this.

File: content/cloudflare-one/policies/filtering/dns-policies/_index.md [hunk 3]
```diff
@@ -245,18 +248,22 @@ Use this selector to filter based on the country where the query arrived to Gate
...
+## Comparison Operators
 
-{{<render file="gateway/_operators.md">}}
+{{<render file="gateway/_comparison-operators.md">}}
 
 ## Value
 
 {{<render file="gateway/_value.md">}}
+
+## Logical Operators
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/dns-policies/_index.md]:
```suggestion
## Logical operators
```

File: content/cloudflare-one/policies/filtering/dns-policies/_index.md [hunk 4]
```diff
@@ -245,18 +248,22 @@ Use this selector to filter based on the country where the query arrived to Gate
...
+| UI name   | API example                  |
+| --------- | ---------------------------- |
 | Source IP | `dns.src_ip == 198.51.100.0` |
 
 ### Users
 
 {{<render file="gateway/_users.md">}}
 
-## Operators
+## Comparison Operators
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/dns-policies/_index.md]:
```suggestion
## Comparison operators
```

File: content/cloudflare-one/policies/filtering/dns-policies/_index.md [hunk 5]
```diff
@@ -245,18 +248,22 @@ Use this selector to filter based on the country where the query arrived to Gate
 
 Use this selector to apply DNS policies to a specific source IP address that queries arrive to Gateway from — for example, this could be the WAN IP address of the stub resolver used by an organization to send queries upstream to Gateway.
 
-| UI name | API example  |
-| -- | -- |
+| UI name   | API example                  |
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/dns-policies/_index.md:251]: Thanks for fixing the table formatting!

File: content/cloudflare-one/_partials/gateway/_logical-operators.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+<div class="special-class" markdown="1">
+
+To evaluate multiple conditions in an expression, select the **Add** logical operator. These expressions can be compared further with the **Or** logical operator.
+
+| Operator | Meaning                                       |
+| -------- | --------------------------------------------- |
+| And      | match all of the conditions in the expression |
+| Or       | match any of the conditions in the expression |
+
+The **Or** operator will only work with conditions in the same expression group. For example, you cannot compare conditions in **Traffic** with conditions in $1.
```
Comment (ranbel) [inline content/cloudflare-one/_partials/gateway/_logical-operators.md:18]: How does the user know whether conditions are in the same expression group?

File: content/cloudflare-one/policies/filtering/http-policies/_index.md [hunk 7]
```diff
@@ -15,12 +15,15 @@ Install the <a href="/cloudflare-one/connections/connect-devices/warp/user-side-
 
 HTTP policies allow you to intercept all HTTP and HTTPS requests and either block, allow, or override specific elements such as websites, IP addresses, and file types. HTTP policies operate on Layer 7 for all TCP (and [optionally UDP](/cloudflare-one/policies/filtering/initial-setup/http/#1-connect-to-gateway)) traffic sent over ports 80 and 443.
 
-An HTTP policy consists of an **Action** as well as a logical expression that determines the scope of the policy. To build an expression, you need to choose a **Selector** and an **Operator**, and enter a value or range of values in the **Value** field.
+An HTTP policy consists of an **Action** as well as a logical expression that determines the scope of the policy. To build an expression, you need to choose a **Selector** and an **Operator**, and enter a value or range of values in the **Value** field. You can use **And** and **Or** logical operators to evaluate multiple conditions.
 
 - [Actions](#actions)
 - [Selectors](#selectors)
-- [Operators](#operators)
+- [Comparison Operators](#comparison-operators)
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/http-policies/_index.md]: same comments as above regarding sentence case

File: content/cloudflare-one/policies/filtering/network-policies/_index.md [hunk 8]
```diff
@@ -15,12 +15,15 @@ To enable this feature, download and deploy the [WARP client](/cloudflare-one/co
...
 
-A network policy consists of an **Action** as well as a logical expression that determines the scope of the action. To build an expression, you need to choose a **Selector** and an **Operator**, and enter a value or range of values in the **Value** field.
+A network policy consists of an **Action** as well as a logical expression that determines the scope of the action. To build an expression, you need to choose a **Selector** and an **Operator**, and enter a value or range of values in the **Value** field. You can use **And** and **Or** logical operators to evaluate multiple conditions.
 
 - [Actions](#actions)
 - [Selectors](#selectors)
-- [Operators](#operators)
+- [Comparison Operators](#comparison-operators)
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/network-policies/_index.md]: same comments as above regarding sentence case

**Review by ranbel — APPROVED** [review]

File: content/cloudflare-one/_partials/gateway/_logical-operators.md (hunk 6 above)
Comment (maxvp) [reply content/cloudflare-one/_partials/gateway/_logical-operators.md:18]: The UI has titles that separate each group.
image

**Pull request merged**